### ISSUE TITLE
[bug] TestReductionsAutomatic#testIrregularSize03 is failing on all backends

### ISSUE THREAD

**Issue by andrii0lomakin** [beehive-lab/TornadoVM#534]
### Describe the bug 
TestReductionsAutomatic#testIrregularSize03 is failing on all backends

### How To Reproduce

` tornado-test -V uk.ac.manchester.tornado.unittests.reductions.TestReductionsAutomatic`

Result is:
```
Test: class uk.ac.manchester.tornado.unittests.reductions.TestReductionsAutomatic
        Running test: testIrregularSize01        ................  [PASS]
        Running test: testIrregularSize02        ................  [PASS]
        Running test: testIrregularSize03        ................  [FAILED]
```

### Expected behavior 

Test is passed

### Computing system setup (please complete the following information):
 - OS: Windows, Ubuntu 22.04
 - OpenCL and Driver versions -  OpenCL version 3.0 , driver version - 23.43.27642.52
 - If applicable, PTX and CUDA Driver versions - driver version 560.81, cuda version 12.6
 - If applicable, Level Zero & SPIR-V Versions - 1.9.3
 - TornadoVM commit id - `35e7e550c`

**Comment by jjfumero** [comment]
Yes, I think this was introduced in the latest integration with Graal. We will take a look,

**Comment by jjfumero** [comment]
Test now passing. and the fix was merged in #540 

```bash
tornado-test -V uk.ac.manchester.tornado.unittests.reductions.TestReductionsAutomatic

WARNING: Using incubator modules: jdk.incubator.vector

Test: class uk.ac.manchester.tornado.unittests.reductions.TestReductionsAutomatic
        Running test: testIrregularSize01        ................  [PASS] 
        Running test: testIrregularSize02        ................  [PASS] 
        Running test: testIrregularSize03        ................  [PASS] 
        Running test: testIrregularSize04        ................  [FP64 UNSUPPORTED FOR CURRENT DEVICE] 
Test ran: 4, Failed: 0, Unsupported: 1

```
